packages/samples/common-types/openapi/v5/networksecurityperimeter.json

```diff
@@ -0,0 +1,518 @@
+          "format": "arm-id",
+          "description": "The fully qualified Azure resource ID of the subscription e.g. ('/subscriptions/00000000-0000-0000-0000-000000000000')"
+        }
+      }
+    },
+    "CreatedByType": {
+      "type": "string",
+      "description": "The type of identity that created the resource.",
+      "enum": [
+        "User",
+        "Application",
+        "ManagedIdentity",
+        "Key"
+      ],
+      "x-ms-enum": {
+        "name": "CreatedByType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "User",
+            "value": "User"
+          },
+          {
+            "name": "Application",
+            "value": "Application"
+          },
+          {
+            "name": "ManagedIdentity",
+            "value": "ManagedIdentity"
+          },
+          {
+            "name": "Key",
+            "value": "Key"
+          }
+        ]
+      }
+    },
+    "IssueType": {
+      "type": "string",
+      "description": "Type of issue",
+      "enum": [
+        "Unknown",
+        "ConfigurationPropagationFailure",
+        "MissingPerimeterConfiguration",
+        "MissingIdentityConfiguration"
+      ],
+      "x-ms-enum": {
+        "name": "IssueType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Unknown",
+            "value": "Unknown",
+            "description": "Unknown issue type"
+          },
+          {
+            "name": "ConfigurationPropagationFailure",
+            "value": "ConfigurationPropagationFailure",
+            "description": "An error occurred while applying the network security perimeter (NSP) configuration."
+          },
+          {
+            "name": "MissingPerimeterConfiguration",
+            "value": "MissingPerimeterConfiguration",
+            "description": "A network connectivity issue is happening on the resource which could be addressed either by adding new resources to the network security perimeter (NSP) or by modifying access rules."
+          },
+          {
+            "name": "MissingIdentityConfiguration",
+            "value": "MissingIdentityConfiguration",
+            "description": "An managed identity hasn't been associated with the resource. The resource will still be able to validate inbound traffic from the network security perimeter (NSP) or matching inbound access rules, but it won't be able to perform outbound access as a member of the NSP."
+          }
+        ]
+      }
+    },
+    "NetworkSecurityPerimeter": {
+      "type": "object",
+      "description": "Information about a network security perimeter (NSP)",
+      "properties": {
+        "id": {
+          "type": "string",
+          "format": "arm-id",
+          "description": "Fully qualified Azure resource ID of the NSP resource",
+          "x-ms-arm-id-details": {
+            "allowedResources": [
+              {
+                "type": "Microsoft.Network/networkSecurityPerimeters"
```

Comment:
There is a diff here 
<img width="1237" alt="image" src="https://github.com/user-attachments/assets/61ad1c79-517b-44f1-9389-af7498182ce1">
good to investigate which one is correct

Reply:
type is the correct one ([AutoRest Extensions for OpenAPI 2.0 | autorest (azure.github.io)](https://azure.github.io/autorest/extensions/#x-ms-arm-id-details)) thanks for your help!!!